packages/testing/README.md

```diff
@@ -93,7 +93,7 @@ Already part of `yo open-wc:testing`
   },
   ```
 
-For more details pls see [testing-karma-bs](https://open-wc.org/recommendations/testing-karma-bs.html)
+For more details pls see [testing-karma-bs](https://open-wc.org/testing/testing-karma-bs.html)
```

Comment:
For more details, please see [testing-karma-bs](https://open-wc.org/testing/testing-karma-bs.html)